test/test_waveform.py

```diff
@@ -34,9 +34,8 @@
 
 _scheme, _context = parse_args_all_schemes("Waveform")
 
-failing_wfs = ['PhenSpinTaylor', 'PhenSpinTaylorRD', 'EccentricTD',
-              'SpinDominatedWf', 'EOBNRv2HM_ROM', 'EOBNRv2_ROM', 'EccentricFD',
-              'NR_hdf5', 'TEOBResum_ROM', 'Lackey_Tidal_2013_SEOBNRv2_ROM']
+# We only check a few as some require auxiliary files
+good_waveforms = ['IMRPhenomD', 'TaylorF2', 'SEOBNRv4', 'SpinTaylorT5']
```

Comment:
Would be good to add a couple more into here, especially the most used things for PE. E.g. PhenomHM, PhenomPv[2,3] (PhenomX depending on what lalsuite we have), SEOBNRv4P (although it may not pass all the tests)

Reply:
I'll try to add the phenom ones in. I know the PhenomXPHM though doesn't work as the EOL python2.7 release of lalsuite doesn't include it. Once we drop python2 support though, I agree. I'll check on SEOBNRv4P